pytensor/tensor/math.py

```diff
@@ -600,7 +808,9 @@ def max_and_argmax(a, axis=None, keepdims=False):
     axis = check_and_normalize_axes(a, axis)
     if len(axis) == 0:
         axis = list(range(a.type.ndim))
-    out, argout = MaxAndArgmax(axis)(a)
```

Comment:
Let's add a FutureWarning deprecating this function and tell people to call max and argmax separately as an alternative